agent/tests/test_lib_aws_sdk_php.c

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "tlib_php.h"
+
+#include "php_agent.h"
+#include "lib_aws_sdk_php.h"
+#include "fw_support.h"
+
+tlib_parallel_info_t parallel_info
+    = {.suggested_nthreads = -1, .state_size = 0};
+
+#if ZEND_MODULE_API_NO > ZEND_7_1_X_API_NO
+
+#define PHP_AWS_CLASS_PREFIX "Supportability/library/aws/aws-sdk-php/"
+#define PHP_AWS_SDK_SERVICE_NAME_METRIC_PREFIX \
+  "Supportability/PHP/AWS/Services/"
```

Comment:
To avoid code duplication, this macro should be defined in `lib_aws_sdk_php.h` and used in both modules: `lib_aws_sdk_php.c` and `test_lib_aws_sdk_php.c`.

Reply:
please see

Reply:
Resolved in [842d437](https://github.com/newrelic/newrelic-php-agent/pull/932/commits/842d4377f3be22d2feca2dd1f32e429ff4bd6a40)